lib/index.js

```diff
@@ -1,6 +1,7 @@
 var Assert = require('assert');
 var Hoek = require('hoek');
 var Joi = require('joi');
+var Schema = require('mongoose').Schema
```

Comment:
Good call. I'm trying to push out an app that uses this, hence I haven't revisited - but I'll knock out those tests when I have some time later this week.